### PR TITLE
Added json file of simplified chinese(zh-cn) translation and dropdown of it

### PR DESCRIPTION
**Pull request by mansijain980**
Fixed : #144 

Now these ML MCQs are available in simplified Chinese as well.

### CODE REVIEW

**Comment by mansijain980** [comment]
@jlooper Here is a PR.

File: quiz-app/src/assets/translations/index.js [hunk 1]
```diff
@@ -3,14 +3,16 @@ import en from './en.json';
...
 
 //export const defaultLocale = 'en';
 
 const messages = {
 	en: en[0],
 	tr: tr[0],
 	fr: fr[0],
-	ja: ja[0]
+	ja: ja[0],
+                   zh-cn: zh-cn[0]
```
Comment (ChungZH) [inline quiz-app/src/assets/translations/index.js:15]: 缩进很奇怪

**Comment by ChungZH** [comment]
你再仔细看看...

**Comment by ChungZH** [comment]
image

File: quiz-app/src/assets/translations/zh-cn.json [hunk 2]
```diff
@@ -0,0 +1,2612 @@
...
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "id": 16,
+            "title": "后勤回归：课后测验",
+            "quiz": [{
+                    "questionText": "海出生是一种类型",
```
Comment (ChungZH) [inline quiz-app/src/assets/translations/zh-cn.json:800]: Seaborn?

Comment (ChungZH) [inline quiz-app/src/assets/translations/zh-cn.json:800]: Seaborn?

File: quiz-app/src/assets/translations/zh-cn.json [hunk 3]
```diff
@@ -0,0 +1,2612 @@
...
+                            "isCorrect": "true"
+                        },
+                        {
+                            "answerText": "输出神经网络交换",
+                            "isCorrect": "false"
+                        }
+                    ]
+                },
+                {
+                    "questionText": "弗拉斯克是如何由它的创造者定义的？",
```
Comment (ChungZH) [inline quiz-app/src/assets/translations/zh-cn.json:869]: Flask? No one knows what you mean.

File: quiz-app/src/assets/translations/zh-cn.json [hunk 4]
```diff
@@ -0,0 +1,2612 @@
...
+                            "isCorrect": "false"
+                        },
+                        {
+                            "answerText": "微型框架",
+                            "isCorrect": "true"
+                        }
+                    ]
+                },
+                {
+                    "questionText": "Python 的泡菜模块是做什么的",
```
Comment (ChungZH) [inline quiz-app/src/assets/translations/zh-cn.json:885]: pickle?

File: quiz-app/src/assets/translations/zh-cn.json [hunk 5]
```diff
@@ -0,0 +1,2612 @@
...
+            "id": 18,
+            "title": "构建 Web 应用程序：讲座后测验",
+            "quiz": [{
+                    "questionText": "我们可以使用哪些工具使用 Python 在网络上托管预先训练的模型？",
+                    "answerOptions": [{
+                            "answerText": "瓶",
+                            "isCorrect": "true"
+                        },
+                        {
+                            "answerText": "滕索弗.js",
```
Comment (ChungZH) [inline quiz-app/src/assets/translations/zh-cn.json:912]: TensorFlow.js?

**Review by ChungZH — COMMENTED** [review]
很多词汇是无法翻译成中文的，不要硬音译。
有些表达比较生硬。
可以考虑重写一遍。
------
Many words cannot be translated into Chinese, such as Flask, TensorFlow, Seaborn... Don't translate them.
I think you should retranslate.

File: quiz-app/src/assets/translations/zh-cn.json (hunk 5 above)
Comment (mansijain980) [reply quiz-app/src/assets/translations/zh-cn.json:912]: Yeah, actually I am confused whether to change them or not. But I translate them. No problem at all.

Comment (ChungZH) [reply quiz-app/src/assets/translations/zh-cn.json:912]: No one can understand.

Comment (mansijain980) [reply quiz-app/src/assets/translations/zh-cn.json:912]: Let me correct typo and all.

**Comment by jlooper** [comment]
thank you for reviewing, @ChungZH - we need to avoid machine translations @mansijain980 if that's what's going on here. I'd rather not have the translation if it's primarily done by machine

File: quiz-app/src/assets/translations/index.js [hunk 6]
```diff
@@ -3,16 +3,19 @@ import en from './en.json';
...
 
 //export const defaultLocale = 'en';
 
 const messages = {
-	en: en[0],
-	tr: tr[0],
-	fr: fr[0],
-	ja: ja[0],
-	it: it[0]
+
```
Comment (ChungZH) [inline quiz-app/src/assets/translations/index.js:12]: empty line

File: quiz-app/src/assets/translations/zh-cn.json [hunk 7]
```diff
@@ -0,0 +1,2612 @@
...
+                        {
+                            "answerText": "比爾蓋茨",
+                            "isCorrect": "false"
+                        },
+                        {
+                            "answerText": "摇晃机器人",
+                            "isCorrect": "false"
+                        }
+                    ]
+                },
```
Comment (ChungZH) [inline quiz-app/src/assets/translations/zh-cn.json:141]: zh-cn???? zh-tw???

File: quiz-app/src/assets/translations/zh-cn.json [hunk 8]
```diff
@@ -0,0 +1,2612 @@
...
+                        {
+                            "answerText": "验证和评估集",
+                            "isCorrect": "false"
+                        }
+                    ]
+                },
+                {
+                    "questionText": "一个常见的命令来启动训练过程中各种ML库是：",
+                    "answerOptions": [{
+                            "answerText": "",
```
Comment (ChungZH) [inline quiz-app/src/assets/translations/zh-cn.json:410]: model.travel

File: quiz-app/src/assets/translations/zh-cn.json [hunk 9]
```diff
@@ -0,0 +1,2612 @@
...
+                }
+            ]
+        },
+        {
+            "id": 11,
+            "title": "准备和可视化回归数据：课前测验",
+            "quiz": [{
+                    "questionText": "以下哪个 Python 模块用于绘制数据的可视化？",
+                    "answerOptions": [{
+                            "answerText": "NumPy的",
```
Comment (ChungZH) [inline quiz-app/src/assets/translations/zh-cn.json:537]: 的？

**Review by ChungZH — CHANGES_REQUESTED** [review]
I think you can close this PR. This is too bad.

**Comment by ChungZH** [comment]
image

Do you know Chinese.....? I really doubt it, not just because of your profile.

**Comment by mansijain980** [comment]
> image
> 
> Do you know Chinese.....? I really doubt it, not just because of your profile.

Yeah @ChungZH I know Russian, Chinese, and English as well.

**Comment by ChungZH** [comment]
> > image
> > Do you know Chinese.....? I really doubt it, not just because of your profile.
> 
> Yeah @ChungZH I know Russian, Chinese, and English as well.

Translated by native Chinese speakers will be better.

**Comment by mansijain980** [comment]
> I think you can close this PR. This is too bad.

Sorry for the Inconvenience.

**Comment by mansijain980** [comment]
> > > image
> > > Do you know Chinese.....? I really doubt it, not just because of your profile.
> > 
> > 
> > Yeah @ChungZH I know Russian, Chinese, and English as well.
> 
> Translated by native Chinese speakers will be better.

Actually, I don't know the natives of there.

**Comment by mansijain980** [comment]
I tried to minimize errors. Please have a look now.

**Comment by ChungZH** [comment]
I don't have time.

**Comment by mansijain980** [comment]
I think it is perfectly fine now

**Comment by ChungZH** [comment]
How can you be so confident? Not a native Chinese speaker and still translating Chinese?
I took a quick look and I can say absolutely NOT FINE.

**Comment by ChungZH** [comment]
As jlooper says: 
> I'd rather not have the translation if it's **primarily done by machine**.

**Comment by jlooper** [comment]
Let’s not argue here. Discussions can take place on the discussion board. I’m going to ask a colleague to review this prior to closing.

**Comment by mansijain980** [comment]
> Let’s not argue here. Discussions can take place on the discussion board. I’m going to ask a colleague to review this before closing.

Sure, Chinese is not my native lang but I learned some languages so I translated them.
I want to contribute to open-source and have a keen interest in ML  so I chose this repo
But it's a kind request to consider this as this is important for now for my academics.

**Comment by jlooper** [comment]
The problem is, low quality or unnecessary contributions just for the sake of contributing makes my life harder (it's like Hacktoberfest); for example right now I'm on vacation and have to take time to deal with this. So I think that it would be better to build your own ML content on your own repo and work through your questions in that way. Thanks, closing this.

**Comment by mansijain980** [comment]
> The problem is, low quality or unnecessary contributions just for the sake of contributing makes my life harder (it's like Hacktoberfest); for example right now I'm on vacation and have to take time to deal with this. So I think that it would be better to build your own ML content on your own repo and work through your questions in that way. Thanks, closing this.

Sorry for the fuss.

**Pull request closed without merging**